### PR TITLE
remove 'Listing' prefix from lists

### PR DESCRIPTION
**Pull request by epinzur**
fixes https://github.com/kaskada-ai/kaskada/issues/336

changes from this:

Screen Shot 2023-05-10 at 7 29 07 AM

to this:
Screen Shot 2023-05-10 at 8 17 26 AM

### CODE REVIEW

File: antora-playbook-PR.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ asciidoc:
     idprefix: ""
     idseparator: "-"
     table-stripes: even
-    listing-caption: Listing
```
Comment (therapon) [inline antora-playbook-PR.yml:43]: This change here will affect all pages under the site. 
I think you can add attributes to the header part of each file to configure that file.

```
:listing-caption: "":  
```

I am not 100% sure we want to just have a number for listings. When you refer to a listing in the text generally you want something to indicate what it is, e.g. Figure 1 or Listing 3 etc. 

I may be too dogmatic from papers/books etc. here. 

What do others think?

Comment (epinzur) [reply antora-playbook-PR.yml:43]: I think we actually want to have the default be that this isn't set, and maybe turn it on for a few pages.  Examples where it is currently isn't being used well:

https://kaskada.io/docs-site/kaskada/main/developing/queries.html#querying-with-the-cli
https://kaskada.io/docs-site/kaskada/main/getting-started/hello-world-cli.html#querying-data
https://kaskada.io/docs-site/kaskada/main/installing.html#using-kaskada-with-python

I can't actually find any page where I think it is being used correctly for source code.

The default in ascii doc is to not have this attribute set for listing & source blocks: https://docs.asciidoctor.org/asciidoc/latest/blocks/add-title/#captioned-titles

Comment (therapon) [reply antora-playbook-PR.yml:43]: I am ok with which ever way we want the default as long as we are consistent. 

By "not used well" I think you mean that if we do not add a caption for a code block nothing is displayed (no number or text). Which will still be the case with the changed proposed here. Some will have a caption and a number but others will not. 

If we have an agreement for the default we then need to crawl the existing docs and add/remove the caption accordingly if we want to be consistent, which seems extra work. We could leave existing pages as is but then there is no consistency, again, and default becomes a matter of personal taste.

**Review by therapon — APPROVED** [review]

File: local-antora-playbook.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ site:
 content:
   sources:
     - url: ../kaskada
-      branches: main
+      branches: docs/listing
```
Comment (therapon) [inline local-antora-playbook.yml:9]: you do not need to add this change :) 
The local version of the file is a good seed that people can change for their local development

Comment (epinzur) [reply local-antora-playbook.yml:9]: this was added by mistake... I'll remove it

**Pull request merged**